pkg/kubelet/allocation/allocation_manager.go

```diff
@@ -696,6 +751,11 @@ func (m *manager) CheckPodResizeInProgress(allocatedPod *v1.Pod, podStatus *kube
 		// (Allocated == Actual) => clear the resize in-progress status.
```

Comment:
Maybe not for this PR (unless you find it quick and easy), but perhaps as a follow-up, I think it would be useful for us to change `TestIsPodResizeInProgress` to `CheckPodResizeInProgress`

Ideally, we could update this test to check:
- For cases where `isPodResizeInProgress` is true, the `InProgress` condition gets set appropriately
- For cases where `isPodResizeInProgress` is false, the `InProgress` condition is cleared and the pod resize complete event is emitted

Reply:
> About this my consideration is, the actuatedResources is updated after the allocatedContainer, and is more reflective of the actual resized resources.

Sorry, I'm not understanding this part. `actuatedResources` should == `allocatedContainer.Resources`; otherwise the resize should still be in progress and is not complete? Are they different for some reason?